pkg/kubelet/cm/cpumanager/policy_static_test.go

```diff
@@ -493,6 +498,45 @@ func TestStaticPolicyAdd(t *testing.T) {
 			expCSet:         cpuset.NewCPUSet(),
 		},
 	}
+	newNUMAAffinity := func(bits ...int) bitmask.BitMask {
+		affinity, err := bitmask.NewBitMask(bits...)
+		if err != nil {
+			panic(err)
+		}
```

Comment:
I would just remove the panic here and ignore the error. If an erorr is returned here as part of test declaration we have bigger issues.

Reply:
Fixed in latest commit